compositor/sessionmanager/sessionmanager.cpp

```diff
@@ -320,9 +285,6 @@ void SessionManager::requestHybridSleep()
     Q_EMIT hybridSleepRequested();
 }
 
-void SessionManager::cancelShutdownRequest()
-{
-    Q_EMIT shutdownRequestCanceled();
-}
+void SessionManager::cancelShutdownRequest() { Q_EMIT shutdownRequestCanceled(); }
 
-#include "moc_sessionmanager.cpp"
+#include "sessionmanager.moc"
```

Comment:
"If a Q_OBJECT or Q_GADGET macro is found in a header file, moc will be run on the file. The result will be put into a file named according to moc_<basename>.cpp. If the macro is found in a C++ implementation file, the moc output will be put into a file named according to <basename>.moc, following the Qt conventions."

this means if you have your declaration in a .h file the moc file is called moc_*.cpp but if both declaration and implementation are in the .cpp file moc wants *.moc like in some plugin.cpp